zap.zsh

```diff
@@ -89,33 +110,28 @@ _zap_update() {
             _pull $plug
             cd "$ZAP_PLUGIN_DIR"
         done
-        cd $pwd > /dev/null 2>&1
     elif [[ $plugin == "0" ]]; then
         cd "$ZAP_DIR"
         _pull 'zap'
-        cd $pwd
     else
-        for plug in $plugins; do
-            selected=$(echo $plug | grep -E "^ $plugin" | awk 'BEGIN { FS = "[ /]" } { print $6 }')
-            cd "$ZAP_PLUGIN_DIR/$selected"
-            _pull $selected
-            cd - > /dev/null 2>&1
-        done
+        cd "$ZAP_PLUGIN_DIR"
+        selected=$(echo $plugins | grep -E "^ $plugin" | awk 'BEGIN { FS = "[ /]" } { print $5 }')
+        cd $selected
+        _pull $selected
     fi
     if [[ $ZAP_CLEAN_ON_UPDATE == true ]]; then
         _zap_clean
     fi
+    cd $pwd > /dev/null 2>&1
 }
 
 _zap_help() {
     cat "$ZAP_DIR/doc.txt"
 }
 
 _zap_version() {
-    ref=$ZAP_DIR/.git/packed-refs
-    tag=$(awk 'BEGIN { FS = "[ /]" } { print $3, $4 }' $ref | grep tags | tail -1)
-    ver=$(echo $tag | cut -d " " -f 2)
-    echo "⚡Zap Version v$ver"
+    release=$(gh release list --repo https://github.com/zap-zsh/zap | grep -E "Latest")
```

Comment:
it's not a good idea to use `gh` for this, not everyone has `gh` installed

Reply:
I couldn't find a better solution, but if anyone has a better idea then it'd be great

Reply:
> I couldn't find a better solution, but if anyone has a better idea then it'd be great

I found one: `curl https://api.github.com/repos/zap-zsh/zap/releases/latest -s | jq .name -r`

Reply:
Thanks. Did that in #77